### PR TITLE
Some UI fixes

### PR DESCRIPTION
**Pull request by WeaxMe**
1. Fixed Markdown support
2. Set property widget order

### CODE REVIEW

File: orienteer-core/src/main/java/org/orienteer/core/component/widget/document/ODocumentPropertiesWidget.java [hunk 1]
```diff
@@ -22,14 +23,17 @@
...
+import java.util.List;
 
 /**
  * Widget to show registered parameters for a document on particular tab
  */
-@Widget(domain="document", id = ODocumentPropertiesWidget.WIDGET_TYPE_ID, autoEnable=true)
+@Widget(
+		domain="document",
+		id = ODocumentPropertiesWidget.WIDGET_TYPE_ID,
+		tab = ODocumentPropertiesWidget.WIDGET_TYPE_ID,
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/component/widget/document/ODocumentPropertiesWidget.java]: As discussed, the widget should be available on any tab, so please do not attach the widget to a particular one.

File: orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java [hunk 2]
```diff
@@ -1,41 +1,57 @@
...
+        }
+        try {
+            MutableDataSet options = new MutableDataSet();
+            options.set(Parser.EXTENSIONS, createExtensions());
+            Parser parser = Parser.builder(options).build();
+            HtmlRenderer renderer = HtmlRenderer.builder(options).build();
+
+            Node node = parser.parse(markDawnModel.getObject());
+            markDownValue = renderer.render(node);
+        } catch (Exception e) {
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java]: Are there more detailed exceptions possible? Maybe make sense to have a more detailed message displayed?

File: orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java [hunk 3]
```diff
@@ -1,41 +1,57 @@
...
+            throw new WicketRuntimeException("Can't use flexmark-java for markups", e);
         }
         return markDownValue;
+    }
 
+    private List<Extension> createExtensions() {
+        List<Extension> extensions = new LinkedList<>();
+        extensions.add(TablesExtension.create());
+        extensions.add(StrikethroughExtension.create());
+        extensions.add(EmojiExtension.create());
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java]: Are Emoji working? Tested? I guess it's related also to images availability through the web.

File: orienteer-core/src/main/resources/log4j.properties [hunk 4]
```diff
@@ -2,7 +2,7 @@ log4j.appender.Stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.Stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.Stdout.layout.conversionPattern=[%.6p] (%d) %.26c{1}: %m%n
 
-log4j.rootLogger=INFO,Stdout
+log4j.rootLogger=DEBUG,Stdout
```
Comment (PhantomYdn) [inline orienteer-core/src/main/resources/log4j.properties]: You can change that locally, but default one it's better to keep as INFO.

File: pom.xml [hunk 5]
```diff
@@ -240,6 +241,11 @@
 				<artifactId>coreui__ajax</artifactId>
 				<version>1.0.6</version>
 			</dependency>
+			<dependency>
+				<groupId>com.vladsch.flexmark</groupId>
+				<artifactId>flexmark-all</artifactId>
+				<version>${flexmark.version}</version>
```
Comment (PhantomYdn) [inline pom.xml]: No need to add all versions as properties. I recommend to have as properties versions only for sets of libs. In this case it's uniq lib usage.

File: orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java (hunk 3 above)
Comment (WeaxMe) [reply orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java]: Yes. Emoji works.

File: orienteer-core/src/main/resources/log4j.properties (hunk 4 above)
Comment (WeaxMe) [reply orienteer-core/src/main/resources/log4j.properties]: Yes. This is my local changes. I added this file to commit accidentally

File: pom.xml (hunk 5 above)
Comment (WeaxMe) [reply pom.xml]: Understood. Thanks

File: orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java (hunk 2 above)
Comment (WeaxMe) [reply orienteer-core/src/main/java/org/orienteer/core/model/MarkDownModel.java]: Will fix this case

File: orienteer-core/src/main/java/org/orienteer/core/component/widget/document/ODocumentPropertiesWidget.java (hunk 1 above)
Comment (WeaxMe) [reply orienteer-core/src/main/java/org/orienteer/core/component/widget/document/ODocumentPropertiesWidget.java]: Understood. Thanks

**Comment by WeaxMe** [comment]
I close PR. I will apply discussed moments and then create new PR

**Pull request closed without merging**